packages/components/addon/components/hds/modal/index.js

```diff
@@ -8,6 +8,14 @@ import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 import { assert } from '@ember/debug';
 import { getElementId } from '@hashicorp/design-system-components/utils/hds-get-element-id';
+import { buildWaiter } from '@ember/test-waiters';
+import { DEBUG } from '@glimmer/env';
```

Comment:
With Ember-CLI's General Purpose ENV flags, this code will get stripped out in production builds

Reply:
`DEBUG` evaluates to `true` in dev/test builds, but `false` in prod builds